pkgs/by-name/om/omnisharp-roslyn/update.sh

```diff
@@ -14,7 +14,10 @@ if [[ "$new_version" == "$old_version" ]]; then
   exit 0
 fi
 
-cd ../../../..
+pushd ../../../..
 update-source-version omnisharp-roslyn "${new_version//v}"
 
 $(nix-build -A omnisharp-roslyn.fetch-deps --no-out-link)
+popd
+
+nixfmt deps.nix
```

Comment:
I think this should be handled elsewhere other than in the update script.

We have #358025 tracking this so I think this commit should be removed and the unformatted file should be committed instead.

Reply:
I guess the nixfmt check will be skipped since deps.nix was already not formatted, but I'm curious how would you do this for new packages, just merge the PR with the check failing?

Reply:
Reverted, thanks for your effort in the linked issue!

Reply:
> I guess the nixfmt check will be skipped since deps.nix was already not formatted, but I'm curious how would you do this for new packages, just merge the PR with the check failing?

Yes, that's the path forward while we don't come up with a long term solution.